internal/hydraclient/hydraclient.go

```diff
@@ -0,0 +1,51 @@
+package hydraclient
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gofrs/uuid"
+
+	"github.com/ory/herodot"
+	hydraclientgo "github.com/ory/hydra-client-go"
+	"github.com/ory/kratos/session"
+)
+
+func GetHydraAdminAPIClient(hydra_admin_url string) hydraclientgo.AdminApi {
+	configuration := hydraclientgo.NewConfiguration()
```

Comment:
Please use the global HTTPClient available in the driver/registry to follow our security requirements and resiliency!